ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx

```diff
@@ -184,6 +184,13 @@ export const delayedAdmissionControlStatusStyle = {
     fgColor: 'text-base-700',
 };
 
+// Special case for Scanner when Sensor is UNHEALTHY or DELAYED.
+export const delayedScannerStatusStyle = {
+    Icon: InfoCircleIcon,
+    bgColor: 'bg-base-200',
+    fgColor: 'text-base-700',
```

Comment:
This is used when the **sensor** status is delayed/unhealthy, not **scanner**. 
When the sensor is unavailable, we can't distribute information about the status of the scanner, so we color it gray.
If the scanner is not available by itself, we color the status in red or green as in other cases.

TBH this was copy-pasted from the collector/admission controller health view, and it is probably worth using the same "delayed" style for all 3 cases, since they are exactly the same.